gdcdictionary/schemas/subject.yaml

```diff
@@ -151,15 +151,22 @@ properties:
     description: >
       First three digits of the resident's zipcode.
     type: string
-    
+
  # Prop originated from SSR project
   version:
     description: >
       The data version as determined by the data submitter.
-    type: string   
+    type: string
 
 
   studies:
     $ref: "_definitions.yaml#/to_many"
   projects:
     $ref: "_definitions.yaml#/to_one_project"
+
+# Prop originated for Longcovid and Monkeypox data
+
+  unique_de-id_identifier:
```

Comment:
looks like property names cannot contain `-`:
```
Validating 'Subject'
ERROR
'unique_de-id_identifier' does not match any of the regexes: '^[_a-zA-Z0-9]*$'
```

Reply:
changed `unique_de-id_identifier` to `unique_de_id_identifier`